eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -722,7 +722,7 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
         case Failure(cause) => handleLocalError(cause, d, Some(fee))
       }
 
-    case Event(c@CMD_SIGN, d: DATA_NORMAL) =>
+    case Event(c: Command, d: DATA_NORMAL) if c == CMD_SIGN =>
```

Comment:
This one was very strange, I have no idea why previous syntax doesn't work. cc @sstone